components/MeetingsList.js

```diff
@@ -1,22 +1,67 @@
+import { useState } from 'react';
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Button,
+} from '@mui/material';
+import Link from 'next/link';
 import styled from 'styled-components';
 
 import Card from '../components/Card';
 
 function MeetingsList({ meetingsList }) {
   return (
-    <StyledMeetingsList>
-      {meetingsList.map(meeting => {
-        return <Card meeting={meeting} key={meeting.id} />;
-      })}
-    </StyledMeetingsList>
+    <>
+      <StyledMeetingsList>
+        {meetingsList.map(meeting => {
+          return (
+            <Link href={`/meetings/${meeting.id}`} key={meeting.id}>
+              <StyledLink>
+                <StyledAnchor>
+                  <Card meeting={meeting} />
+                </StyledAnchor>
```

Comment:
```suggestion
                 <Card meeting={meeting} />
```

StyledAnchor is an element without a purpose. It has no styling and therefore it's not needed.